cli/central-client/src/main/java/org/ballerinalang/central/client/util/Utils.java

```diff
@@ -0,0 +1,471 @@
+     * @param outputStream  home repo balo file output stream
+     * @param totalSizeInKB response input stream size in kb
+     * @param fullPkgName   full package name, <org-name>/<pkg-name>:<pkg-version>
+     * @param outStream     Output print stream
+     * @param logFormatter  log formatter
+     */
+    private static void writeAndHandleProgress(InputStream inputStream, FileOutputStream outputStream,
+            long totalSizeInKB, String fullPkgName, PrintStream outStream, LogFormatter logFormatter) {
+        int count;
+        byte[] buffer = new byte[1024];
+
+        try (ProgressBar progressBar = new ProgressBar(fullPkgName + " [central.ballerina.io -> home repo] ",
+                totalSizeInKB, 1000, outStream, ProgressBarStyle.ASCII, " KB", 1)) {
+            while ((count = inputStream.read(buffer)) > 0) {
+                outputStream.write(buffer, 0, count);
+                progressBar.step();
+            }
+        } catch (IOException e) {
+            outStream.println(logFormatter.formatLog(fullPkgName + "pulling the package from central failed"));
+        } finally {
+            outStream.println(logFormatter.formatLog(fullPkgName + " pulled from central successfully"));
+        }
+    }
+
+    /**
+     * Create nightly build meta file.
+     *
+     * @param nightlyBuildMetaFilePath nightly build meta file path
+     * @param logFormatter             log formatter
+     */
+    private static void createNightlyBuildMetaFile(Path nightlyBuildMetaFilePath, LogFormatter logFormatter) {
+        try {
+            Files.createFile(nightlyBuildMetaFilePath);
+        } catch (Exception e) {
+            throw ErrorUtil.createCommandException(
+                    logFormatter.formatLog("error occurred while creating nightly.build file."));
+        }
+    }
+
+    /**
+     * Convert string to URL.
+     *
+     * @param url string URL
+     * @return URL
+     */
+    public static URL convertToUrl(String url) {
+        try {
+            return new URL(url);
+        } catch (MalformedURLException e) {
+            throw ErrorUtil.createCommandException(e.getMessage());
+        }
+    }
+
+    /**
+     * Initialize SSL.
+     */
+    public static void initializeSsl() {
+        try {
+            SSLContext sc = SSLContext.getInstance(SSL);
+            sc.init(null, trustAllCerts, new java.security.SecureRandom());
+            HttpsURLConnection.setDefaultSSLSocketFactory(sc.getSocketFactory());
+        } catch (NoSuchAlgorithmException | KeyManagementException e) {
+            throw ErrorUtil.createCommandException("initializing SSL failed: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Set request method of the http connection.
+     *
+     * @param conn   http connection
+     * @param method request method
+     */
+    public static void setRequestMethod(HttpURLConnection conn, RequestMethod method) {
+        try {
+            conn.setRequestMethod(getRequestMethodAsString(method));
```

Comment:
We can replace `getRequestMethodAsString(method)` with `method.name()` as I know

Reply:
Added